### PR TITLE
[Mellanox] Fix 'sensors.conf' mapping for MSN3420

### PR DESCRIPTION
**Pull request by shlomibitton**
**- Why I did it**
Fix 'sensors.conf' mapping for SN3420

**- How I did it**
Edit the 'sensors.conf'.

**- How to verify it**
Run 'sensors' command on Mellanox MSN3420 platform.

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: device/mellanox/x86_64-mlnx_msn3420-r0/sensors.conf [hunk 1]
```diff
@@ -33,8 +33,8 @@ bus "i2c-5" "i2c-1-mux (chan_id 4)"
         label curr1 "PMIC-1 COMEX 1.8V Rail Curr (out)"
         label curr2 "PMIC-1 COMEX 1.05V Rail Curr (out)"
     chip "tps53679-i2c-*-71"
-        label in1 "PMIC-2 PSU 12V Rail (in)"
-        label in2 "PMIC-2 PSU 12V Rail (in)"
+        label in1 "PMIC-2 PSU 12V Rail (in1)"
+        label in2 "PMIC-2 PSU 12V Rail (in2)"
```
Comment (nazariig) [inline device/mellanox/x86_64-mlnx_msn3420-r0/sensors.conf:37]: @shlomibitton why do we need in1/in2. IMHO, it doesn't provide to much info for user. What was the motivation for renaming? Please elaborate.

Comment (shlomibitton) [reply device/mellanox/x86_64-mlnx_msn3420-r0/sensors.conf:37]: on sonic-mgmt we can't have the same label for both in1 and in2. Doing so will fail the sensor output parser, so we have to change the labels like this.
to be consistent and match the labels I have changed them on sonic-buildimage as well.

Comment (nazariig) [reply device/mellanox/x86_64-mlnx_msn3420-r0/sensors.conf:37]: @shlomibitton thanks for explanation.

**Comment by liat-grozovik** [comment]
@abhi-shukla can you please take to 201911?

**Pull request merged**